### PR TITLE
feat: add instructions to check for other dependencies in guide_github

### PR DESCRIPTION
**Pull request by TsubasaK111**
@shunjikonishi Minor fixes, review me!

### CODE REVIEW

File: source/docs/ja/guide_github.md [hunk 1]
```diff
@@ -7,8 +7,8 @@
 - <a href="https://git-scm.com/" target="_blank">Git</a> (Gitの使い方・ドキュは[ここ](https://git-scm.com/book/ja/v2)。)  
 - <a href="https://nodejs.org/en/download/" target="_blank">Node.js/npm</a>
 - <a href="https://app.code-check.io/settings/social" target="_blank">GitHubアカウントをcodecheck.ioに接続</a>  
-- 最後に `npm install codecheck` ってCLIで唱えて  
-codecheck CLI を召喚しておいてね。
+- 最後に `npm install codecheck` ってCLIで唱えてcodecheck CLI を召喚しておいてね。
```
Comment (shunjikonishi) [inline source/docs/ja/guide_github.md]: `npm install codecheck -g`

**Comment by shunjikonishi** [comment]
@TsubasaK111 fix me
`-g` is required to install codecheck as a global module.

Ah, of course!

**Comment by TsubasaK111** [comment]
@shunjikonishi Finished, sir! Review me!

**Comment by shunjikonishi** [comment]
LGTM

**Pull request merged**